WooCommerce/Classes/ViewRelated/Orders/Order Creation/OrderForm.swift

```diff
@@ -121,34 +118,36 @@ struct OrderFormPresentationWrapper: View {
                           presentProductSelector: presentProductSelector)
             }, secondaryView: { isShowingProductSelector in
                 if let productSelectorViewModel = viewModel.productSelectorViewModel {
-                    ProductSelectorView(configuration: productSelectorConfiguration,
+                    ProductSelectorView(configuration: .loadConfiguration(for: horizontalSizeClass),
                                         source: .orderForm(flow: flow),
                                         isPresented: isShowingProductSelector,
                                         viewModel: productSelectorViewModel)
                     .sheet(item: $viewModel.productToConfigureViewModel) { viewModel in
                         ConfigurableBundleProductView(viewModel: viewModel)
                     }
                 }
-            }, onSizeClassChange: { sizeClass in
-                productSelectorConfiguration = ProductSelectorConfiguration.loadConfiguration(for: sizeClass)
```

Comment:
Thanks for updating this, I see now that, as you mentioned, the environment key is just a wrapper around the system-provided `horizontalSizeClass`, so we could have used that directly. If I had checked that initially, rather than focusing in our custom key, I could have avoided some pain 😅 

💯